src/tests/ftest/container/open.py

```diff
@@ -170,3 +166,14 @@ def test_container_open(self):
             print(traceback.format_exc())
             self.assertEqual(expected_result, RESULT_FAIL,
                              result_messages[test_case][3])
+
+        # Calling container.open with invalid pool handle or UUID results in
+        # setting them in the underlying DaosContainer's member. When we try to
+        # destroy it with API in tearDown, it'll pass in the invalid
+        # handle/UUID member to the API and causes an error. Set
+        # self.container = None so that container destroy doesn't occur during
+        # tearDown. Note that we shouldn't set None when open succeeds because
+        # in that case, the container needs to be properly destroyed for the
+        # pool destroy process to work during tearDown.
+        if not self.container[0].opened or not self.container[1].opened:
+            self.container = None
```

Comment:
Shouldn't we just reset the handle/UUID to the correct value(s) so that `tearDown()` works correctly?

Reply:
Done